scripts/move_to_spark_1.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set +x
+
+grep -q "spark2" pom.xml
```

Comment:
Not entirely sure I understand your objection here. We run the same command in both `move_to_spark_1` and `move_to_spark_2` but we check for different exit codes. In the Spark 1 script, we check for exit code 1 which indicates that "spark2" was not found in pom.xml, while in the Spark 2 script, we check for exit code 0, which indicates that "spark2" was found in pom.xml.